### DISCUSSION TITLE
Why safelist does'n accept regexp array

### DISCUSSION THREAD

**Discussion by charlietyn** [unocss/unocss#1095] [Ideas]
Why safelist does'n accept regexp array, in tailwind regexp is accepted in unocss no

**Comment by antfu** [comment]
Everything in UnoCSS is dynamic and on-demand, meaning you have an infinite possibility to match with if you use regex. 

See also https://windicss.org/guide/extractions.html#safelist

**Reply by charlietyn** [reply]
Thanks 👍
